application/src/main/kotlin/auth/AuthServiceImpl.kt

```diff
@@ -45,7 +45,7 @@ class AuthServiceImpl(
             .withIssuer(issuer.value)
             .withAudience(audience.value)
             .withClaim("email", email)
-            .withExpiresAt(Date(System.currentTimeMillis() + expirationTime)) // Token expires in 1 minute
+            .withExpiresAt(Date(System.currentTimeMillis() + expirationTime)) // Token expires in 1 hour
```

Comment:
6_000_000 milliseconds = 100 minutes 
```suggestion
            .withExpiresAt(Date(System.currentTimeMillis() + expirationTime)) // Token expires in 100 minutes
```